polaris-react/src/components/Frame/Frame.scss

```diff
@@ -149,6 +149,11 @@ $sidebar-breakpoint: 1200px;
   width: 100%;
   height: $top-bar-height;
 
+  #{$se23} {
+    z-index: var(--p-z-index-10);
```

Comment:
Considering there is a patch already in web for the TopBar. I think we should consider extending the patch in web to fix this in the short term (i.e. before editions) or adjust Polaris to remove the patch that exists within web today.

The use cases we need to be considerate of are any popovers, sidekick, settings, settings with sidekick, settings with search, bulk editor, and bulk editor with popovers. There are likely others, but these would be the major pieces we'd need to vet with adjusting the z-index.

Reply:
@mattkubej let me know what you think about this approach in web https://github.com/Shopify/web/pull/99117